dfastmi/io.py

```diff
@@ -196,354 +171,6 @@ def get_text(key: str) -> List[str]:
-):
-    chapters = [k for k in config.keys()]
-    checksum = ""
-    checkval = 1
-    for chapter in chapters:
-        keys = [k for k in config[chapter].keys()]
-        for key in keys:
-            str = config[chapter][key]
-            if chapter == "General" and key == "checksum":
-                checksum = str
-            else:
-                checkval = zlib.adler32(str.encode("utf-8"), checkval) & 0xffffffff
-    #print("Expected checksum: ", checkval)
-    if checksum == "":
-        log_text("checksum", dict = {"filename": filename})
-    else:
-        checkval2 = int(checksum)
-        if checkval2 != checkval:
-            raise Exception("Checksum mismatch: configuration file {} has been modified!".format(filename))
-
-
-def collect_values1(
-    config: configparser.ConfigParser,
-    branches: List[str],
-    nreaches: List[int],
-    key: str,
-    default: Optional[float] = None,
-) -> List[List[float]]:
-    """
-    Collect river parameter data from river configuration object.
-
-    This routines collects entries of type float.
-
-    Arguments
-    ---------
-    config : configparser.ConfigParser
-        The dictionary containing river data.
-    branches : List[str]
-        The list of river branches. The length of this list is nBranches.
-    nreaches : List[int]
-        The number of reaches per river branch. The length of this list is nBranches.
-    key : str
-        The name of the parameter for which the values are to be retrieved.
-
-    Raises
-    ------
-    Exception
-        If the number of values read from the file doesn't match 1.
-
-    Returns
-    -------
-    data : List[List[float]]
-        A list of lists. Each list contains per reach within the corresponding
-        branch one float.
-    """
-    try:
-        g_val = config["General"][key]
-    except:
-        g_val = ""
-
-    all_values = []
-    for ib in range(len(branches)):
-        branch = branches[ib]
-        try:
-            b_val = config[branch][key]
-        except:
-            b_val = g_val
-
-        values_per_branch = []
-        for i in range(nreaches[ib]):
-            stri = str(i + 1)
-            try:
-                val = config[branch][key + stri]
-            except:
-                val = b_val
-            if val == "" and default is not None:
-                fval = default
-            else:
-                try:
-                    vals = tuple(float(x) for x in val.split())
-                except:
-                    vals = ()
-                if len(vals) != 1:
-                    raise Exception(
-                        'Reading {} for reach {} on {} returns "{}". Expecting {} values.'.format(
-                            key, i+1, branches[ib], val, 1
-                        )
-                    )
-                fval = vals[0]
-            values_per_branch.append(fval)
-
-        all_values.append(values_per_branch)
-
-    return all_values
-
-
 def collect_int_values1(
```

Comment:
This should also be in riverobject I think, test related to this can then also be moved.

Reply:
discussed will be tackeled in another issue